frontend/src/components/Common/Banner.tsx

```diff
@@ -11,13 +10,6 @@ import close from '../../assets/svg/common/close.svg'
 interface Props extends PropsFromRedux {}
 
 const Banner: FC<Props> = (props) => {
-  const location = useLocation();
-  const history = useHistory();
-
-  function redirect(site: string) {
```

Comment:
Let's keep this redirect function and use it for future external links. Using the redirect route allows us to view analytics of users clicking on the link.
You can add a comment noting to use this function in the future for external links instead of `href`.